### PR TITLE
[json/cs] JSON translation to cs_CZ

### PR DESCRIPTION
**Pull request by vojtasvoboda**
JSON translation to cs_CZ

### CODE REVIEW

File: cs-cz/json.html.markdown [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+language: json
+filename: learnjson.json
```
Comment (vendethiel) [inline cs-cz/json.html.markdown]: please add a `-cz` suffix to the filename

**Comment by vojtasvoboda** [comment]
Suffix added

**Comment by vendethiel** [comment]
Mh, it seems the property is actually duplicated

**Comment by vojtasvoboda** [comment]
Oh, you are right, duplicated attribute removed.

**Comment by vendethiel** [comment]
Thanks, and sorry for the delay!

**Pull request merged**